var/spack/repos/builtin/packages/cpuinfo/package.py

```diff
@@ -25,33 +26,12 @@ class Cpuinfo(CMakePackage):
     depends_on("cmake@3.5:", type="build")
     depends_on("ninja", type="build")
 
-    resource(
-        name="googletest",
-        url="https://github.com/google/googletest/archive/release-1.10.0.zip",
-        sha256="94c634d499558a76fa649edb13721dce6e98fb1e7018dfaeba3cd7a083945e91",
-        destination="deps",
-        placement="googletest",
-    )
-    resource(
-        name="googlebenchmark",
-        url="https://github.com/google/benchmark/archive/v1.2.0.zip",
-        sha256="cc463b28cb3701a35c0855fbcefb75b29068443f1952b64dd5f4f669272e95ea",
-        destination="deps",
-        placement="googlebenchmark",
-    )
-
     generator = "Ninja"
 
     def cmake_args(self):
         return [
-            self.define(
-                "GOOGLETEST_SOURCE_DIR", join_path(self.stage.source_path, "deps", "googletest")
-            ),
-            self.define(
-                "GOOGLEBENCHMARK_SOURCE_DIR",
-                join_path(self.stage.source_path, "deps", "googlebenchmark"),
-            ),
-            self.define("CPUINFO_BUILD_UNIT_TESTS", self.run_tests),
-            self.define("CPUINFO_BUILD_MOCK_TESTS", self.run_tests),
-            self.define("CPUINFO_BUILD_BENCHMARKS", self.run_tests),
```

Comment:
Decided to remove these. They aren't technically tests since they change what gets installed, and the things that get installed are broken anyway.